tbx/static_src/sass/components/_grid.scss

```diff
@@ -157,7 +157,8 @@
         }
     }
 
-    &__promo {
+    &__promo,
+    &__event {
```

Comment:
looking at figma I don't think the event block goes right to the edge.

Reply:
In [figma](https://www.figma.com/file/QdMXBHr57mN1svsaNQ7Ggj/Torchbox-2024-(Master)?type=design&node-id=3568-50588&mode=design&t=IxBffvPEEbVs8T4S-4), it was equal to the promo block, so I made it the same width.